RunningOrder/Shared/Representables/FocusableNSTextField.swift

```diff
@@ -31,9 +30,18 @@ struct FocusableTextField: NSViewRepresentable {
         return textField
     }
 
-    func updateNSView(_ nsView: NSTextField, context: Context) {
+    func updateNSView(_ nsView: FocusableNSTextField, context: Context) {
         nsView.stringValue = value
         nsView.placeholderString = placeholder
+        if !isFocused && context.coordinator.oldFocused == true {
```

Comment:
you can add some comment here to explain the nested conditions

Reply:
comments added